### PR TITLE
People with the voracious quirk always appear fat

### PR DESCRIPTION
**Pull request by Mickyan**
A bit of extra flavor, you can now fatshame your co-workers
:cl:
tweak: Picking the voracious quirk now always makes you appear chubby when examined!
/:cl:

### CODE REVIEW

**Comment by Ty-the-Smonk** [comment]
what happens if your nutrition level is fat and you have the trait?

**Comment by Time-Green** [comment]
> what happens if your nutrition level is fat and you have the trait?

Nothing, it checks for either condition in the same place.

**Comment by HomocidlePancakes** [comment]
Fat sprites when?

**Comment by RaveRadbury** [comment]
I'm not a fan of this because I use it on a character that I wouldn't want to have constantly seen as fat. If this goes through I'll be dropping it from the character.

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -214,7 +214,7 @@
 
 	if(nutrition < NUTRITION_LEVEL_STARVING - 50)
 		msg += "[t_He] [t_is] severely malnourished.\n"
-	else if(nutrition >= NUTRITION_LEVEL_FAT)
+	else if(nutrition >= NUTRITION_LEVEL_FAT || (HAS_TRAIT(src, TRAIT_VORACIOUS)))
 		if(user.nutrition < NUTRITION_LEVEL_STARVING - 50)
 			msg += "[t_He] [t_is] plump and delicious looking - Like a fat little piggy. A tasty piggy.\n"
```
Comment (Fikou) [inline code/modules/mob/living/carbon/human/examine.dm]: what the FUCK

**Comment by ATH1909** [comment]
What happens if they're starving?

**Comment by Mickyan** [comment]
I forgot putting quirks on examine isn't good for identification purposes, it was a fun thought

File: code/modules/mob/living/carbon/human/examine.dm (hunk 1 above)
Comment (itseasytosee) [reply code/modules/mob/living/carbon/human/examine.dm]: Time to roll back the commits and find out who the fuck made this.

Comment (Anonmare) [reply code/modules/mob/living/carbon/human/examine.dm]: It's from ~2012, ancient curse

**Pull request closed without merging**